packages/cli/src/setup-gcloud.ts

```diff
@@ -35,16 +106,54 @@ const siginGcloudAndSetIAM = async (projectId: string, pwd: string) => {
     'roles/serviceusage.serviceUsageAdmin',
   ];
   const serviceAccount = await checkServiceAccountExists(projectId);
-  if (!serviceAccount) {
+  const description = 'A service account with permission to deploy Cloud Commerce from the GitHub repository to Firebase';
+  const displayName = 'Cloud Commerce GH Actions';
+
+  if (!serviceAccount && haveGcloud) {
     await $`gcloud iam service-accounts create ${serviceAccountId} \
-      --description="A service account with permission to deploy Cloud Commerce from the GitHub repository to Firebase" \
-      --display-name="Cloud Commerce GH Actions"`;
+      --description="${description}" --display-name="${displayName} "`;
```

Comment:
```suggestion
      --description=${description} --display-name=${displayName}`;
```

https://github.com/google/zx/blob/main/docs/quotes.md